assets/package.json

```diff
@@ -25,7 +25,7 @@
     "watch:css": "nodemon -w src/styles -e scss -x \"npm run build:css\"",
     "watch:fonts": "nodemon -w node_modules/uswds/src/fonts -x \"npm run build:fonts\"",
     "watch:images": "nodemon -w src/img -x \"npm run build:images\"",
-    "watch:js": "rollup -c --watch"
+    "watch:js": "mkdir -p dist/scripts && cp node_modules/date-time-format-timezone/build/browserified/date-time-format-timezone-complete-min.js dist/scripts && rollup -c --watch"
```

Comment:
You should not have removed the directory creation and cp from the build:js. Please restore it. This is necessary so when the artifiact is build it contains this file. The watch script is not run during the build process on Jenkins. It is strictly for development.